python/cudf/cudf/core/reshape.py

```diff
@@ -122,9 +122,10 @@ def concat(objs, axis=0, join="outer", ignore_index=False, sort=None):
 
     Parameters
     ----------
-    objs : list of DataFrame, Series, or Index
+    objs : list or dictionary of DataFrame, Series, or Index
```

Comment:
perhaps we can tighten up the two suggestions you've made in a separate issue/PR since it seems out of scope to handle that here for this change, what do you think?

Reply:
Yes, that makes sense to postpone. As you saw, the keys are not sorted, so we're doing the right thing here.